### ISSUE TITLE
include index.js to help with creating includePaths?

### ISSUE THREAD

**Issue by pdehaan** [eduardoboucas/include-media#108]
Something like https://github.com/ranjandatta/node-normalize-scss/blob/master/index.js

This would allow us to easily add **include-media** to the Sass includePath without needing to reach into the node_modules directory.

---

I was able to get this working, but I had to use the following code:

```scss
@import "../node_modules/include-media/dist/include-media";

$hidpi: "retina2x";

#foo {
  @include media($hidpi) {
    content: 'Device Pixel Ratio of at least 1.5';
  }
}
```

**Comment by eduardoboucas** [comment]
Not sure I understand what this does exactly. Can you provide some more examples or, even better, attempt a PR? :smile:

**Comment by drazik** [comment]
Yeah that can be very useful. If I understand it well, it could make us able to import include-media this way :

```scss
@import "include-media";

@include media(">small") {
    //....
}
```

@eduardoboucas, are you interested ?
@pdehaan, do you want to attempt a PR ? If not, I can do it.

**Comment by eduardoboucas** [comment]
Sure, I'd love to see a PR.

**Comment by drazik** [comment]
I think it can be closed since #110 has been merged

**Comment by eduardoboucas** [comment]
Indeed. Thanks again @drazik!
